app/Routes.php

```diff
@@ -308,6 +308,15 @@ function () use ( $ffFactory ) {
 			}
 		)->bind( 'contact' );
 
+		$app->get(
+			'faq',
+			function () use ( $ffFactory ) {
+				return $ffFactory->getLayoutTemplate( 'page_layouts/faq.html.twig' )->render(
```

Comment:
Now that it's a standalone route we should move the template out of `page_layouts`.